general/forall.hpp

```diff
@@ -87,58 +87,98 @@ void OmpWrap(const int N, HBODY &&h_body)
 
 
 /// RAJA Cuda backend
-#if defined(MFEM_USE_RAJA) && defined(RAJA_ENABLE_CUDA)
+#if defined(MFEM_USE_RAJA) && defined(RAJA_DEVICE_ACTIVE)
```

Comment:
`RAJA_ENABLE_DEVICE` ?

Reply:
``RAJA_DEVICE_ACTIVE`` checks we if are compiling with a device compiler (hipcc, or nvcc), ``RAJA_ENABLE_DEVICE`` has been removed I believe.